src/html/index.html

```diff
@@ -55,13 +55,13 @@
 
     <img class="logo-small" src="assets/img/krig-logomark.svg">
 
-    <div class="sticky--left social-nav">
-      <a class="social-nav__item social-nav__item--facebook" href="https://www.facebook.com/krig.collective/"><img src="assets/img/facebook.svg"></a>
-      <a class="social-nav__item social-nav__item--github" href="https://github.com/krig-collective"><img src="assets/img/github.svg"></a>
-      <a class="social-nav__item social-nav__item--instagram" href="https://www.instagram.com/krig_collective/"><img src="assets/img/instagram.svg"></a>
-      <a class="social-nav__item social-nav__item--twitter" href="https://twitter.com/krig_io"><img src="assets/img/twitter.svg"></a>
-      <a class="social-nav__item social-nav__item--email" href="mailto:webmasterkrig.io"><img src="assets/img/mail.svg"></a>
-    </div>
+    <ul class="sticky--left social-nav">
```

Comment:
Will these behave like links or will we need to add JS to handle it? :)

Reply:
I'll add link elements in the li items :)